onecodex/api.py

```diff
@@ -103,6 +105,15 @@ def __init__(
         )
         self._client._fetch_schema(cache_schema=cache_schema)
         self._session = self._client.session
+
+        # Set backoff / retry strategy for 429s
+        retry_strategy = Retry(
```

Comment:
I think the retry code in `onecodex/lib/upload.py` can be removed now that we have retries being set here